.buildkite/test-pipeline.yaml

```diff
@@ -466,9 +458,11 @@ steps:
   - vllm/
   - tests/compile
   commands:
-  - pytest -v -s compile/test_basic_correctness.py
-  - pytest -v -s compile/test_multimodal_compile.py
-  - pytest -v -s compile/piecewise/
+  # Run smoke tests under fullgraph directory, except test_full_graph.py
+  # as it is a heavy test that is covered in other steps.
+  # Use `find` to launch multiple instances of pytest so that
+  # they do not suffer from https://github.com/vllm-project/vllm/issues/21073
```

Comment:
Can you add this comment to other places that use `find`?

Reply:
Added. I also filed a more appropriate issue for the problem with more context:
https://github.com/vllm-project/vllm/issues/28965